store/copr/mpp.go

```diff
@@ -260,13 +295,21 @@ func (m *mppIterator) establishMPPConns(bo *tikv.Backoffer, req *kv.MPPDispatchR
 		return
 	}
 
-	// TODO: cancel the whole process when some error happens
 	for {
 		err := m.handleMPPStreamResponse(bo, resp, req)
 		if err != nil {
+			m.cancelMppTasks(bo, taskMeta)
```

Comment:
the cancel job will take some time. We'd better send error at first, then cancel the tasks.

Reply:
ok